sdk/js/src/Client.ts

```diff
@@ -26,6 +27,12 @@ import { dvcDefaultLogger } from './logger'
 import { DVCLogger } from '@devcycle/types'
 import { StreamingConnection } from './StreamingConnection'
 
+type variableUpdatedHandler = (key: string, variable: DVCVariable<DVCVariableValue> | null) => void
+type featureUpdatedHandler = (key: string, feature: DVCFeature) => void
```

Comment:
what happens here when you get removed from a feature?

Reply:
good point, it's undefined too - switched it to null